Tests/DependencyInjection/AbstractDoctrineExtensionTest.php

```diff
@@ -399,6 +399,22 @@ public function testEntityManagerMemcacheMetadataCacheDriverConfiguration()
         ));
     }
 
+    public function testEntityManagerRedisMetadataCacheDriverConfiguration()
```

Comment:
Can you change the name of the method so that it mention the database key of the configuration being used.